### ISSUE TITLE
Flutter sdk integration 

### ISSUE THREAD

**Issue by Allan-Nava** [heroiclabs/nakama#593]
Is interesting integrate with flutter

**Comment by Allan-Nava** [comment]
I'm trying to create a sdk for flutter: https://github.com/Allan-Nava/nakama-flutter

**Comment by erlend-sh** [comment]
Interesting. Do you have a specific use case that you're building this for?

**Comment by Allan-Nava** [comment]
> Interesting. Do you have a specific use case that you're building this for?

Less pain to develop a mobile application with one single code base for iOS and Android

**Comment by erlend-sh** [comment]
Are you building an application in Flutter that will use Nakama for some online functionality?

**Comment by Allan-Nava** [comment]
> Are you building an application in Flutter that will use Nakama for some online functionality?

Yes! actually I'm using the native kotlin and swift code , but the idea is to create a simple chat application in flutter

**Comment by obrunsmann** [comment]
We are also interested in using Nakama in production with Flutter. Flutter becomes more and more interesting also for 2D-Game development (e.g. https://flame-engine.org).
Thank you @Allan-Nava for your first try on a Dart SDK! 👍

**Comment by Allan-Nava** [comment]
> We are also interested in using Nakama in production with Flutter. Flutter becomes more and more interesting also for 2D-Game development (e.g. https://flame-engine.org).
> Thank you @Allan-Nava for your first try on a Dart SDK! 👍

I have problem with [generate.sh](https://github.com/Allan-Nava/nakama-flutter/blob/main/protos/generate.sh)

Coud you please help me?

Thanks

**Comment by obrunsmann** [comment]
Yes we are still puzzling over a build script - but we are getting much closer and not much is missing.
I'll let you know when I have a working version.

**Comment by Allan-Nava** [comment]
> Yes we are still puzzling over a build script - but we are getting much closer and not much is missing.
> I'll let you know when I have a working version.

Okay let me know ☺️

**Comment by Allan-Nava** [comment]
https://github.com/Allan-Nava/nakama-flutter/issues/3#issuecomment-850533815 Need help to implement the flutter sdk

**Comment by obrunsmann** [comment]
Hey Guys, we finished a pure dart version for Nakama: https://pub.dev/packages/nakama
Not yet supporting 100% of the gRPC API but more is coming day per day :)

**Comment by duck-dev-go** [comment]
+1

**Comment by vmtest888** [comment]
+1

**Comment by novabyte** [comment]
I'll close this issue as completed. Anyone who's interested to follow development of our Dart/Flutter SDK can keep a watch on this repository: https://github.com/heroiclabs/nakama-dart
